src/mmap.rs

```diff
@@ -149,6 +149,7 @@ impl<B: Bitmap> GuestRegionMmap<B> {
     }
 }
 
+#[cfg(not(feature = "xen"))]
```

Comment:
Yes, some documentation will be good. Daemons need to have `xen` feature too now and build separately for both. Though I have now added a UNIX mapping model with xen, where it can also do UNIX type mappings, kvm i.e.

Reply:
Ok. I think this may become awkward when packaging stuff, but I will leave this to the maintainers to decide. Code complexity wise I personally do not see a huge difference between conditional compilation and checks at runtime.

Reply:
I have updated README with this inforamtion now.